rs/ic-observability/multiservice-discovery/src/metrics.rs

```diff
@@ -0,0 +1,68 @@
+use opentelemetry::{
+    global,
+    metrics::{Counter, UpDownCounter},
+    KeyValue,
+};
+
+const NETWORK: &str = "network";
+
+#[derive(Clone)]
+pub struct MSDMetrics {
+    pub definitions: UpDownCounter<i64>,
+    pub running_definition_metrics: RunningDefinitionsMetrics,
+}
+
+impl Default for MSDMetrics {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl MSDMetrics {
+    pub fn new() -> Self {
+        let meter = global::meter("axum-app");
+        let definitions = meter
+            .i64_up_down_counter("msd.definitions")
+            .with_description("Total number of definitions that multiservice discovery is scraping")
```

Comment:
```suggestion
            .with_description("Total number of definitions that multiservice discovery is serving")
```
targets ==> scraping
definitions ==> serving
or am I wrong?

Reply:
Definition == network, we scrape networks
1 network has many targets, we serve targets on /targets